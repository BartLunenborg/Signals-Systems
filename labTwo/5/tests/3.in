2: [1,42]
10: [1,42,1,42,1,42,10,52,10,52]
