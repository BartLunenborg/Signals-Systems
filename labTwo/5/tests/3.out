9: [1765,84,1765,84,1765,462,2194,472,2194]
